javascript/javascript3/week1/weather.js

```diff
@@ -0,0 +1,111 @@
+const main = document.querySelector('main');
+const cityInput = document.getElementById('city');
+const para = document.querySelector('p');
+const getData = document.getElementById('weather-btn');
+
+getData.addEventListener('click', getWeatherData);
+
+//Fire Event from Enter
+cityInput.addEventListener('keyup', event => {
+    if (event.key === 'Enter') {
+        getWeatherData (); 
+    }
+})
+// Reach API
+function getWeatherData (){
+    const cityName = cityInput.value.toLowerCase();
+    const api = "https://api.openweathermap.org/data/2.5/weather?q=";
+    const key = "&appid=0f5d1d4ac034115ef7ae1bcdcada3527";
+    const metric = "&units=metric";
+    const url = api + cityName + key + metric;
+    console.log(url);
+        if (cityName === "") {
+            para.innerText = 'Type a city name first.';
+        } else {
+            para.innerText = "";
+            fetchData(url)
+        };
+}
+// Fetch Weather Data for all
+function fetchData (url) {
+    fetch(url)
+    .then(result => {
+        if (result.ok) {
+          return result.json(); 
+        } else {
+            para.innerText = "Incorrect city name";
+        }
+      })
+    .then (data => {
+        showWeather(data);
+   })
+}
+// Change Background Images
+function changeBgImg(data){
+    let mainData = data.weather[0].main;
+        mainData === "Clear"?
```

Comment:
use a switch statement here, instead of a clever ternary operator ?:?:?:?:?: It is only good practice to use a ternary operator if you have 1 simple if / then / else